wordpress-mobile/WordPress-iOS
# Notifications: remove notification for deleted Comment

Ref: #17790, https://github.com/wordpress-mobile/WordPress-iOS/pull/18010#pullrequestreview-891446056

When a comment is deleted from My Site Comments, the corresponding notification is removed from the Notifications list. In split view, if the currently selected notification is removed, the first available notification is selected and displayed.

To test:
- In split view, go to Notifications. 
- Select a Comment notification.
- Go to My Sites tab > Comments.
- Delete the comment for the selected notification.
- Return to Notifications.
- Verify:
  - The notification for the deleted comment is removed.
  - The first available notification is selected and the details are shown.
  - The table is responsive.
- Disable `notificationCommentDetails`.
- Repeat the steps above.


## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
